### ISSUE TITLE
SoapySDR remote questions

### ISSUE THREAD

**Issue by jfath** [rsadsb/dump1090_rs#49]
I'm working on getting dump190_rs and radar running using SoapySDR remote with an Airspy.  It seems to be running, but I'm not actually seeing any airplanes in radar.  One thing I discovered is that Airspy has three different gain controls:
    LNA gain range: [0, 15] dB
    MIX gain range: [0, 15] dB
    VGA gain range: [0, 15] dB

My questions:
1)  It seems I can only specify one gain key in the custom-config file unless I'm missing something?  I can hack the Airspy soapy driver to add an "ALL" that will set them all to the same value, but judging by other signals, I think setting just VGA should bring in at least strong signals.
2) While dump1090_rs is running, it prints a steady stream of characters (mostly 'S', some '0') to stdout.  Is that expected?  Do the characters mean anything?

Thanks for these projects - I was very happy to find an ads-b project that supports remote hw as my sdr and antenna are located in a barn away from the e noise of my house.

**Comment by wcampbell0x2a** [comment]
These are StreamStatusIndicators: https://github.com/pothosware/SoapySDR/wiki/StreamStatusIndicators.

Thanks! I enjoyed writing it.

**Comment by wcampbell0x2a** [comment]
Yes, I've only seen `O` when running `dump1090_rs` in debug mode instead of release mode, but if you are doing that it's most likely network related.
